deepchem/utils/data_utils.py

```diff
@@ -267,9 +268,7 @@ def load_sdf_files(input_files: List[str],
                 for task in tasks:
                     df_row.append(mol.GetProp(str(task)))
 
-            conf = mol.GetConformer()
-            positions = conf.GetPositions()
-            pos_x, pos_y, pos_z = zip(*positions)
+            pos_x, pos_y, pos_z = get_xyz_from_mol(mol).tolist()
```

Comment:
`get_xyz_from_mol` will return an array of shape `(N, 3)`. The above line will raise a `ValueError: too many values to unpack (expected 3)`. Maybe you should transpose the array to shape (3, N) before converting it to list and then extract pos_x, pos_y, pos_z.